src/chainparams.cpp

```diff
@@ -156,6 +156,7 @@ class CMainParams : public CChainParams {
         vSeeds.push_back(CDNSSeedData("dashjr.org", "dnsseed.bitcoin.dashjr.org"));
         vSeeds.push_back(CDNSSeedData("bitcoinstats.com", "seed.bitcoinstats.com"));
         vSeeds.push_back(CDNSSeedData("xf2.org", "bitseed.xf2.org"));
+        vSeeds.push_back(CDNSSeedData("bitcoin.jonasschnelli.ch", "seed.bitcoin.jonasschnelli.ch", true));
```

Comment:
@cdecker: yes. Its the same seeder software and the same data. If you query the seeder DNS with a "x." in front of the query domain it will xor the output. https://github.com/sipa/bitcoin-seeder/pull/24